services/rest/service.go

```diff
@@ -26,6 +26,8 @@ func NewRestService(c collection.Collector) *Service {
 
 	go reportConnectionMetrics(*wh.Table())
 
+	go websocket.AckHandler(websocket.AckChan)
```

Comment:
Is single go routine scalable & performant?
Should we consider enabling multiple goroutines handling the Acks, as there could potentially be 100s and thousands of batches for 10K connections handled in an instance.

Reply:
If this is efficient, then let's stick to just 1 goroutine @AkbaraliShaikh

Reply:
You can choose to resolve this comment

Reply:
A single go-routine is efficient as per the perf test.